src/repositorios/colllection.test.js

```diff
@@ -0,0 +1,162 @@
+
+
+describe(
+    "Test CollectionModel",
+    () => {
+
+        /*
+            Test For Create Collection 
+        */
+
+        test(
+            "create_collection",
+            () => {
+                return CollectionDOA.create_collection({
+                    name: "Jest",
+                    slang: "jest",
+                    public: true,
+                    description: "test with jest",
+                    owner: 1
+                })
+                .then(
+                    data => {
+                        expecte(data)
+                            .toMatchObject(
+                                {
+                                    id: expect.any(Number),
+                                    name: "Jest",
+                                    slang: "jest",
+                                    public: true,
+                                    description: "test with jest",
+                                    owner: 1
+                                }  
+                            );
+                    }
+                )
+            }
+        );
+
+        test(
+            "create_collection without name",
+            () => {
+                return CollectionDOA.create_collection({
+                    slang: "jest",
+                    public: true,
+                    description: "test with jest",
+                    owner: 1
+                })
+                .then(
+                    data => {
+                        expecte(data).toBeNull();
+                    }
+                )
+            }
+        );
+
+        test(
+            "create_collection without slang",
+            () => {
+                return CollectionDOA.create_collection({
+                    name: "Jest",
+                    public: true,
+                    description: "test with jest",
+                    owner: 1
+                })
+                .then(
+                    data => {
+                        expecte(data).toBeNull();
+                    }
+                )
+            }
+        );
+
+        test(
+            "create_collection without description",
+            () => {
+                return CollectionDOA.create_collection({
+                    name: "Jest",
+                    public: true,
+                    slang: "jest",
+                    owner: 1
+                })
+                .then(
+                    data => {
+                        expecte(data).toBeNull();
+                    }
+                )
+            }
+        );
+
+        test(
+            "create_collection without public",
+            () => {
+                return CollectionDOA.create_collection({
+                    name: "Jest",
+                    slang: "jest",
+                    description: "test with jest",
+                    owner: 1
+                })
+                .then(
+                    data => {
+                        expecte(data).toBeNull();
```

Comment:
The default should be public and the missing field should go to the default one.